src/sys.c

```diff
@@ -497,269 +491,262 @@ int reset_disk(void) {
         return -1;
     }
     return fat_flush(&fat);
 }
 
-void seekdir(DIR *dirp, long loc){
+void seekdir(DIR* dirp, long loc) {
     rewinddir(dirp);
-    dir_desc_t *ddesc = (void*)dirp;
-    for(long i = 0;i < loc;i++){
+    dir_desc_t* ddesc = (void*)dirp;
+    for (long i = 0; i < loc; i++) {
         fat_entry_t ent;
-        do{
-            if(fat_dir(&ddesc->base.file, &ent)){
+        do {
+            if (fat_dir(&ddesc->base.file, &ent)) {
                 return;
             }
-        } while(ent.attributes & FAT_ATTRIBUTE_LABEL);
+        } while (ent.attributes & FAT_ATTRIBUTE_LABEL);
         ddesc->pos++;
     }
 }
 
-long telldir(DIR *dirp){
-    dir_desc_t *ddesc = (void*)dirp;
+long telldir(DIR* dirp) {
+    dir_desc_t* ddesc = (void*)dirp;
     return ddesc->pos;
 }
 
-void rewinddir(DIR *dirp){
-    dir_desc_t *ddesc = (void*)dirp;
+void rewinddir(DIR* dirp) {
+    dir_desc_t* ddesc = (void*)dirp;
     fat_rewind(&ddesc->base.file);
     ddesc->pos = 0;
 }
 
-int mkdir(const char *path, mode_t mode){
-    if(init_fat()){
+int mkdir(const char* path, mode_t mode) {
+    if (init_fat()) {
         return -1;
     }
     uint8_t attr = FAT_ATTRIBUTE_DIRECTORY;
-    if(!(mode & S_IRUSR)){
+    if (!(mode & S_IRUSR)) {
         attr |= FAT_ATTRIBUTE_HIDDEN;
     }
-    if(!(mode & S_IWUSR)){
+    if (!(mode & S_IWUSR)) {
         attr |= FAT_ATTRIBUTE_READONLY;
     }
-    const char *tail;
-    fat_path_t *fp = get_origin(path, &tail);
-    if(fat_create(&fat, fat_path_target(fp), tail, attr, NULL)){
+    const char* tail;
+    fat_path_t* fp = get_origin(path, &tail);
+    if (fat_create(&fat, fat_path_target(fp), tail, attr, NULL)) {
         return -1;
     }
     return fat_flush(&fat);
 }
 
-int rmdir(const char *path){
-    if(init_fat()){
+int rmdir(const char* path) {
+    if (init_fat()) {
         return -1;
     }
     fat_entry_t ent;
-    if(wd_find(path, &ent)){
+    if (wd_find(path, &ent)) {
         return -1;
     }
-    if(!(ent.attributes & FAT_ATTRIBUTE_DIRECTORY)){
+    if (!(ent.attributes & FAT_ATTRIBUTE_DIRECTORY)) {
         errno = ENOTDIR;
         return -1;
     }
-    if(entry_access(&ent, 1)){
+    if (entry_access(&ent, 1)) {
         return -1;
     }
-    if(fat_remove(&ent)){
+    if (fat_remove(&ent)) {
         return -1;
     }
     return fat_flush(&fat);
 }
 
-int stat(const char *path, struct stat *buf){
-    if(init_fat()){
+int stat(const char* path, struct stat* buf) {
+    if (init_fat()) {
         return -1;
     }
     fat_entry_t ent;
-    if(wd_find(path, &ent)){
+    if (wd_find(path, &ent)) {
         return -1;
     }
-    if(buf){
+    if (buf) {
         ent_stat(&ent, buf);
     }
     return 0;
 }
 
-int lstat(const char *path, struct stat *buf){
-    return stat(path, buf);
-}
+int lstat(const char* path, struct stat* buf) { return stat(path, buf); }
 
 static int _err = 0;
 
-int *__errno() {
-    return &_err;
-}
+int* __errno() { return &_err; }
```

Comment:
I think `AllowShortFunctionsOnASingleLine: false` is probably more consistent with what it was before